### PR TITLE
colin commit

### PR DESCRIPTION
**Pull request by colin-code-au**
colin's pull request

### CODE REVIEW

**Review by colin-code-au — COMMENTED** [review]
It is too simple, add more text.

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+colin test
```
Comment (colin-code-au) [inline test.txt:1]: better to add a space in the beginning of the line

**Comment by RainieRainie** [comment]
GoodDay~Looks Great~
Approved~hahaha

File: test.txt (hunk 1 above)
Comment (RainieRainie) [reply test.txt:1]: Add comments before coding should be better

**Pull request merged**